AzureStack/2015-04-01/swagger/GalleryAdmin.json

```diff
@@ -0,0 +1,502 @@
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ProviderGalleryRegistrations"
+        ],
+        "description": "Deletes a specific gallery registration",
+        "operationId": "ProviderGalleryRegistrations.Delete",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "galleryItemName",
+            "in": "path",
+            "description": "Identity of the gallery item. Includes publisher name, item name and may include version separated by period character.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.gallery.providers/galleryItems": {
+      "get": {
+        "tags": [
+          "ProviderGalleryRegistrations"
+        ],
+        "description": "Lists gallery registrations",
+        "operationId": "ProviderGalleryRegistrations_List",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ResourceDefinition"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+     "GalleryItemUriPayload": {
+      "description": "Gallery item URI, Points to the http or https uri where the galery package is located, typically a blob uri",
+      "type": "object",
+      "properties": {
+        "GalleryItemUri": {
+          "description": "Gets or sets the gallery item URI.",
```

Comment:
Remove the phrase Gets or sets... AutoRest will automatically add it based on the mutability of the property.